lib/widgets/drop_down_menu/drop_down_menu.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+class DropDownMenu extends StatefulWidget {
+  const DropDownMenu({super.key});
+
+  @override
+  State<DropDownMenu> createState() => _DropDownMenuState();
+}
+
+class _DropDownMenuState extends State<DropDownMenu> {
+  String dropdownvalue = 'GDSC';
+
+  // List of items in our dropdown menu
+  var items = [
```

Comment:
CustomDropDown means it is supposed to be used just like DropdownButton with some simplifications like of styling

The idea for a separate widget is great but hardcoding the items is not good
Try passing it to the class from outside